contribs/src/main/java/com/netflix/conductor/contribs/kafka/resource/handlers/ResourcesLoader.java

```diff
@@ -66,7 +64,7 @@ public void locateResources() {
             while (urls.hasMoreElements()) {
                 readJarFile(urls.nextElement());
             }
-        } catch (IOException ex) {
+        } catch (final IOException ex) {
             logger.error("Can not locate Resources for Kafka Listener", ex);
```

Comment:
be consistent with error logging, ie pass the ex.getMessage() as a param to the log as with other logging

Reply:
Corrected